panel/src/components/Forms/Previews/ToggleFieldPreview.vue

```diff
@@ -1,34 +1,28 @@
 <template>
-	<!-- eslint-disable vue/no-mutating-props -->
 	<div class="k-toggle-field-preview">
-		<k-input
+		<k-toggle-input
 			:text="text"
 			:value="value"
-			type="toggle"
 			@input="$emit('input', $event)"
 		/>
 	</div>
 </template>
 
 <script>
+import FieldPreview from "@/mixins/forms/fieldPreview.js";
+
 export default {
-	props: {
-		field: Object,
-		value: Boolean,
-		column: Object
-	},
+	mixins: [FieldPreview],
 	computed: {
 		text() {
-			return this.column.text !== false ? this.field.text : null;
+			return this.column?.text !== false ? this.field?.text : null;
```

Comment:
Since you remove the `?` in other previews, why adding them here?

Reply:
Makes sense to remove them here as well.